src/index.js

```
@@ -138,59 +164,39 @@ import { on, emit } from "./sdk-wrapper";
         if (!Array.isArray(pathsCTA) || pathsCTA.length === 0) {
-          // console.log('getPathsCTA - no valid paths_cta in config, checking URL...');
-
           const urlParam = new URLSearchParams(window.location.search).get('paths_cta');
-
           try {
             // Try to decode and parse the URL parameter as JSON
             const parsed = JSON.parse(decodeURIComponent(urlParam));
-            //console.log('getPathsCTA - parsed from URL:', parsed);
-
             if (Array.isArray(parsed)) {
               pathsCTA = parsed;
```

Biome (1.9.4)

[error] Can't assign pathsCTA because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)